### DISCUSSION TITLE
Equivalent to Style.Resources?

### DISCUSSION THREAD

**Discussion by sam-wheat** [microsoft/microsoft-ui-xaml#8223] [Q&A] [answered]
In WPF I created styles as shown below.  This allows child components to be implicitly styled.  Is there an equivalent in WinUI?

    <Style TargetType="local:TotalHeaderCell"  BasedOn="{StaticResource BaseTotalCellStyle}">
        <Style.Resources>
            <Style TargetType="TextBlock" BasedOn="{StaticResource LeftTextBlockStyle}"></Style>
        </Style.Resources>
    </Style>

**Comment by castorix** [comment, marked as answer]
There was a discussion about that : [Discussion: Add a new Style.Resources property to simplify Lightweight Styling #3939](https://github.com/microsoft/microsoft-ui-xaml/issues/3939)

**Reply by sam-wheat** [reply]
Thank you!
